app/assets/stylesheets/application.scss

```
@@ -51,4 +51,9 @@ div.text-card-container {
   height: 370px;
   max-width: 376px;
   margin: 0 auto;
+}
+
+.form-wrapper{
+  width:70%;
+  max-width:350px;
```

width を % で指定されるなら，`max-width` は必要ないですね。
あと，下側の空きが小さすぎますので，margin-bottomを入れた方がよいと思います。

```suggestion
  width: 70%;
```

---

WEBサイトの幅だとwidth: 70%;だと大きすぎるので、max-width:350px;を指定しました。実際のアプリも検索窓の大きさが一定に近い為、一定になるようにとしたのですが、ご意見お伺いしたいです。

---

> WEBサイトの幅だとwidth: 70%;だと大きすぎるので、max-width:350px;を指定しました。
すみません，関係ありましたね。残しておいて下さい。

---

再度、ご確認頂き有難うございます！
インデントとmarginの修正プルリクをこれから出しますので、お願い致します。